src/components/Reports/ProjectReport/ProjectReport.jsx

```diff
@@ -29,9 +34,28 @@ export const ProjectReport = ({ match }) => {
     }
   }, []);
 
+  useEffect(() => {
+    wbs.WBSItems.forEach(wbs => {
+      dispatch(fetchAllTasks(wbs._id));
+    });
+  }, [wbs]);
+  
+  useEffect(() => {
+    if (tasks.taskItems.length > 0) {
+      setHoursCommitted(tasks.taskItems.reduce((total, task) => total + task.estimatedHours, 0));
```

Comment:
I recently joined the community and am not yet fully acquainted with the project. Out of curiosity, do the estimated hours for all tasks represent committed hours? I intuitively feel that we might need to use `timeEntries` for this calculation.

Please correct me if I'm wrong, thank you!